docs/pages/components/custom-provider/en-US/index.md

```diff
@@ -35,7 +35,7 @@ return (
 import { CustomProvider } from 'rsuite';
 
 return (
-  <CustomProvider themes="dark">
+  <CustomProvider theme="dark">
```

Comment:
Thank you for fixing the typo in the docs. You'd better fix the other docs as well.
https://github.com/rsuite/rsuite/blob/main/docs/pages/components/custom-provider/zh-CN/index.md